vhs-frontend/src/pages/Explore.tsx

```diff
@@ -37,12 +37,9 @@ export const Explore = () => {
       {vhsList.map(
         (item) =>
           item.id && (
-            <VhsThumbnail
-              key={item.id}
-              image={item.thumbnail ? `/${item.thumbnail}` : placeholder}
-              vhsId={item.id}
-              vhsTitle={item.title}
-            />
+            <Link to={`${item.id}`} key={item.id}>
+              <VhsCard key={item.id} image={item.thumbnail} vhsId={item.id} vhsTitle={item.title} />
```

Comment:
When you're mapping, only the top level component needs to have the key prop. Here, VhsCard doesn't need it.

Reply:
Also, I always put the key prop first so it's more easily visible. This way I know if it's missing faster than if I put it at a random position.